addons/l10n_sa/views/report_invoice.xml

```diff
@@ -20,11 +20,17 @@
         </xpath>
         <xpath expr="//t[@t-set='address']" position="after">
             <t t-set="information_block">
-                <p>
-                    <img t-if="o.l10n_sa_qr_code_str"
-                         style="display:block;"
-                         t-att-src="'/report/barcode/?barcode_type=%s&amp;value=%s&amp;width=%s&amp;height=%s'%('QR', quote_plus(o.l10n_sa_qr_code_str), 200, 200)"/>
-                </p>
```

Comment:
Although I visually appreciate your fix (which should ultimately be validated by an expert or PO), it is breaking the XPath and potentially impacting the client's customizations. Could you revise it to ensure this issue is avoided?

Reply:
Hello @yosa-odoo,

Thank you for your review.

I’m not sure I fully understand how this change could break the XPath. This template inherits from `l10n_gcc_invoice.arabic_english_invoice`, not from `account.report_invoice_document` like the standard invoice templates.

The proposed fix is inspired by how the shipping address is handled in `account.report_invoice_document`, where it is placed in the information block. However, since the information block is already occupied by the QR code in the Saudi Arabia case, I added the new element next to it. As far as I can tell, this approach shouldn’t cause any issues.

https://github.com/odoo/odoo/blob/1b999e4b358eabe80cb3d636cd7bc348c5b85a1b/addons/account/views/report_invoice.xml#L11-L16

If I missed something or misunderstood your concern, could you please clarify how this might break the XPath?